src/Features/Core/Portable/PdbSourceDocument/IPdbSourceDocumentLoaderService.cs

```diff
@@ -10,11 +10,12 @@ namespace Microsoft.CodeAnalysis.PdbSourceDocument
 {
     internal interface IPdbSourceDocumentLoaderService
     {
-        Task<SourceFileInfo?> LoadSourceDocumentAsync(string tempFilePath, SourceDocument sourceDocument, Encoding encoding, TelemetryMessage telemetry, CancellationToken cancellationToken);
+        Task<SourceFileInfo?> LoadSourceDocumentAsync(string tempFilePath, SourceDocument sourceDocument, Encoding encoding, TelemetryMessage telemetry, bool useEasedTimeout, CancellationToken cancellationToken);
```

Comment:
`useExtendedTimeout` perhaps?

Or from the check below, maybe `isFromSourceLinkedProject`?

Similarly on `ShouldCauseTimeoutEase`.  Maybe `fromSourceLinkedProject`?

Reply:
I like useExtendedTimeout, thank you.

I'm not a fan of being specific about source link here, that's an impl detail at this level IMO 😛

Reply:
Changed `ShoudlCauseTimeoutEase` to `FromRemoteLocation`, as a compromise.. and because my original name was truly terrible